cristin-project/src/main/java/no/unit/nva/cristin/projects/update/ProjectPatchValidator.java

```diff
@@ -237,4 +240,15 @@ private void validateExtraLanguages(ObjectNode input) throws BadRequestException
     private BadRequestException notAListOfMapsException() {
         return new BadRequestException(format(NOT_A_VALID_LIST_OF_KEY_VALUE_FIELDS, ALTERNATIVE_TITLES));
     }
+
+    private void validateWebPage(ObjectNode input) throws BadRequestException {
+        if (input.has(WEB_PAGE)) {
```

Comment:
```
        Optional.ofNullable(input.get(WEB_PAGE))
            .map(JsonNode::asText)
            .map(URI::create)
            .orElseThrow(failure -> fieldIsNotUri());
```
If you want to follow the same style as in `CristinProjectBuilder.java`

Reply:
input.has and input.get works a little different. Having something to do with the field present or not, and the field being null or not when present. Tests failed when i tried you example

Reply:
Did you know that ?
Wouldn't that have implications for when we are serializing empty fields to storage.
(we omit them, instead of adding with null)

Reply:
yes i did not it but tested your suggestion just to see if we could simplify. When doing patch requests we have to build the json manually because patch allows us to explicit send a field value as null, so we have to know if the client sends the field at all, sends the field with value null, or field with a regular value

Reply:
Also added a test for checking if field will be sent as null to upstream

Reply:
Maybe something that the whole team should know about ?